modules/spring-config/src/main/java/org/mule/runtime/config/internal/dsl/spring/PropertiesMapBeanDefinitionCreator.java

```diff
@@ -28,20 +28,21 @@
 import org.springframework.beans.factory.support.ManagedList;
 import org.springframework.beans.factory.support.ManagedMap;
 
-class PropertiesMapBeanDefinitionCreator extends BeanDefinitionCreator {
+class PropertiesMapBeanDefinitionCreator extends BeanDefinitionCreator<CreateComponentBeanDefinitionRequest> {
 
   @Override
   boolean handleRequest(Map<ComponentAst, SpringComponentModel> springComponentModels,
-                        CreateBeanDefinitionRequest createBeanDefinitionRequest) {
-    ComponentAst componentModel = createBeanDefinitionRequest.getComponentModel();
-    if (componentModel.getIdentifier().equals(MULE_PROPERTIES_IDENTIFIER)
-        || componentModel.getIdentifier().equals(MULE_PROPERTY_IDENTIFIER)) {
+                        CreateComponentBeanDefinitionRequest createBeanDefinitionRequest) {
+    ComponentAst component = createBeanDefinitionRequest.getComponent();
+    if (component != null
```

Comment:
Inverting this if would allow the happy path to be un-indented (The return false is too far from its cause)